### ISSUE TITLE
Definite assignment assertion on @Input properties

### ISSUE THREAD

**Issue by fireflysemantics** [AckerApple/stripe-angular#20]
Hi - Thanks for this awesome project!  I've been reviewing the code base and just have a few suggestions that I think apply.

### @Input properties

They should not use a definite assignment assertion or `!`.  New angular projects don't allow it.  I double checked this with SO here:

https://stackoverflow.com/questions/59093938/angular-input-property-ending-with-exclamation-point

### ng-container usage

We don't really need an ng-container in the stripe card:

```
<ng-container *ngIf="!StripeScriptTag.StripeInstance">
          <div style="color:red;">Stripe PublishableKey NOT SET. Use method StripeScriptTag.setPublishableKey()</div>
</ng-container>
```

IIUC - This will work just as well:

```
<div *ngIf="!StripeScriptTag.StripeInstance style="color:red;">Stripe PublishableKey NOT SET. Use method StripeScriptTag.setPublishableKey()</div>
```

Thanks again for this sweet project.  I'm learning a lot by studying the code base.

**Comment by AckerApple** [comment]
I agree, I think with most everything you have brought up. However, this library was thrown together out of need and I no longer need it. You are welcome to make a pull request for your refinements. In fact I just accepted a very large pull request #23 plus I upgraded us to Angular 9.

Be well.

**Comment by AckerApple** [comment]
I have changed jobs to where I needed this library again. I have released a new major update. More work to come.

Hopefully next round of updates I can scoop these recommendations up
